src/Debugging-Core/Context.extension.st

```diff
@@ -245,38 +245,46 @@ Context class >> simulatePrimitiveNumber: num with: simulator [
 Context >> stepToHome: aContext [
 	"Resume self until the home of top context is aContext.  Top context may be a block context."
 
-	| home ctxt here error context |
+	| home ctxt here error wrapperContext |
 	here := thisContext.
-	ctxt := self step.
-	ctxt = self ifFalse: [ "Insert ensure and exception handler contexts under aSender"
-		error := nil.
-		context := aContext insertSender: (Context contextOn: UnhandledError do: [ :ex | 
+	error := nil.
+	
+	wrapperContext := aContext insertSender: (Context contextOn: Exception do: [ :ex | 
 				            error
 					            ifNil: [ 
-						            error := ex exception.
-						            ex resumeUnchecked: here jump ]
-					            ifNotNil: [ ex pass ] ]) ].
+						            			error := ex.
```

Comment:
Before this was `ex exception` any idea of why the change?

Reply:
Because before it was an UnhandledError

Reply:
Maybe because ex is an unhandled exception that is an exception refering to the exception that is not handled (and it has that accessor). 
Probably there they have different signaler contexts and the important one is the unhandled exception's context? (but why?)

Reply:
ah sorry I commented and saw your answer after refreshing the page.
So now ex is not the unhandled error anymore?